crowdpact/apps/pact/migrations/0003_auto_20150411_2325.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pact', '0002_auto_20150411_1952'),
```

Comment:
Looks like the datetime the migration was made.  I bet this is a mechanism to avoid migration name collisions.